unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsHUD/ExampleQuests.txt

```diff
@@ -0,0 +1,124 @@
+[
```

Comment:
Can we move this to `TestResources`?

Reply:
Oh, dont mind this file at all. I'm just using it for testing purposes until kernel is ready to send real or mocked data.